pkg/segment/query/processor/inputlookupcommand_test.go

```diff
@@ -0,0 +1,306 @@
+// Copyright (c) 2021-2024 SigScalr, Inc.
+//
+// This file is part of SigLens Observability Solution
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+package processor
+
+import (
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"testing"
+
+	"github.com/siglens/siglens/pkg/config"
+	"github.com/siglens/siglens/pkg/segment/query/iqr"
+	"github.com/siglens/siglens/pkg/segment/structs"
+	"github.com/siglens/siglens/pkg/segment/utils"
+	"github.com/stretchr/testify/assert"
+)
+
+func initTestConfig(t *testing.T) error {
+	runningConfig := config.GetTestConfig(t.TempDir())
+	runningConfig.DataPath = "inputlookup-data/"
```

Comment:
Why are you setting up the test config with a temp dir, but then immediately overriding the name to `inputlookup-data`? Ideally, just use the temp dir. If you need to hardcode the data dir instead, then you should add this code to make sure the dir gets deleted even if the test panics:
```
	t.Cleanup(func() { os.RemoveAll("inputlookup-data/") })
```